app/upgrades/v23/constants.go

```diff
@@ -17,3 +17,12 @@ var Upgrade = upgrades.Upgrade{
 		Deleted: []string{},
 	},
 }
+
+var MigratedIncentiveAccumulatorPoolIDs = map[uint64]struct{}{
+	1423: {},
+	1213: {},
+	1298: {},
+	1297: {},
+	1292: {},
+	1431: {},
+}
```

Comment:
I see that these pools are marked green in the spreadsheet.

Why is pool 1323 green as well despite being migrated in v24?

Reply:
Ah sorry my mistake, it will be migrated and I have highlighted it incorrectly :sweat_smile: